src/common/AdaptiveStream.cpp

```diff
@@ -1212,17 +1238,22 @@ size_t adaptive::AdaptiveStream::getSegmentPos()
 
 bool AdaptiveStream::waitingForSegment(bool checkTime) const
 {
+  if (current_adp_->GetStreamType() == StreamType::SUBTITLE)
+    return false;
+
   if ((m_tree->HasManifestUpdates() || m_tree->HasManifestUpdatesSegs()) && state_ == RUNNING)
   {
     std::lock_guard<adaptive::AdaptiveTree::TreeUpdateThread> lckUpdTree(m_tree->GetTreeUpdMutex());
 
     if (current_rep_ && current_rep_->IsWaitForSegment())
     {
-      return !checkTime ||
-             (current_adp_->GetStreamType() != StreamType::VIDEO &&
-              current_adp_->GetStreamType() != StreamType::AUDIO) ||
-             SecondsSinceUpdate() < 1;
```

Comment:
`SecondsSinceUpdate` uses, above and here should be the cause of the RAI buzz/frezze sound, by debugging this waitingForSegment method, was producing messed up values